recipes/pyamg/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "3.0.2" %}
+
+package:
+    name: pyamg
+    version: {{ version }}
+
+source:
+    fn: pyamg-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pyamg/pyamg-{{ version }}.tar.gz
+    md5: bd024253ebae6f7bf2d89fe32fc5d64a
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - zlib  # [linux]
+    run:
+        - python
+        - numpy x.x
```

Comment:
So, does it link against NumPy or not? If so, it should be a build requirement. If not, the `x.x` should be dropped here.